linq4j/src/test/java/org/apache/calcite/linq4j/tree/TypeTest.java

```diff
@@ -62,4 +83,236 @@
     assertEquals(Object.class, Types.gcd(String.class, int.class));
     java.io.Serializable o = true ? "x" : 1;
   }
+
+  private static void assertFields(List<Field> expected, List<Field> computed) {
+    assertEquals(expected, computed, "Expected field(s) ["
+        + expected + "] but found [" + computed + "]");
```

Comment:
"expected but found" message will be automatically added by `assertEquals`, so please remove it from the message.

assertion message should clarify **why** certain value was expected.